compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5534,6 +5623,8 @@ THRIFT_REGISTER_GENERATOR(
     "                     Enable rethrow of unhandled exceptions and let them propagate further."
     " (Default behavior is to catch and log it.)\n"
     "    java5:           Generate Java 1.5 compliant code (includes android_legacy flag).\n"
+    "    generate_future_iface:\n"
```

Comment:
Can we shorten that to "future_iface"?

Reply:
thanks for the suggestion, updated @Jens-G